app/src/androidTest/java/com/github/HumanLearning2021/HumanLearningApp/view/GoogleSignInWidgetTest.kt

```diff
@@ -63,10 +83,52 @@ class GoogleSignInWidgetTest {
         assertThat(GoogleSignInWidget.isAdmin, equalTo(false))
     }
 
+
+    @Test
+    fun test_signOutIsDisplayed() {
+
+        runBlocking {
+            launchFragment()
+            val firebaseUser = Firebase.auth.signInAnonymously().await().user!!
+            fragment.presenter.onSuccessfulLogin(false)
+            fragment.activity?.runOnUiThread {
+                fragment.updateUi()
+            }
+        }
+        onView(withId(R.id.singOutButton)).check(matches((isDisplayed())))
+
+    }
+
+
+    @Test
+    fun signOutUserSuccess() {
+        runBlocking {
+            launchFragment()
```

Comment:
Usually calling launchFragment in setup is enough.

Reply:
Done :)